autoware_map_msgs/srv/LoadPCDMaps.srv

```diff
@@ -0,0 +1,22 @@
+# Mode definition
+uint8 MODE_PARTIAL_AREA_LOAD=0
+uint8 MODE_DIFFERENTIAL_AREA_LOAD=1
```

Comment:
I think it is better to separate the service type. How about this?
example:
- PartialPointCloudMap.srv
- DifferentialPointCloudMap.srv

Reply:
I see, thanks. That's much easier to understand.
Resolved https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/ce57ce1baab167dc84d36543e38b3b0aa8b72d31